integrations/java/openhouse-java-runtime/src/main/java/com/linkedin/openhouse/javaclient/OpenHouseCatalog.java

```diff
@@ -88,6 +92,8 @@ public class OpenHouseCatalog extends BaseMetastoreCatalog
 
   protected Map<String, String> properties;
 
+  private Cache<TableOperations, FileIO> fileIOCloser;
```

Comment:
these look like temporary duplication of code that will be available in upstream iceberg with version bump. Can we add a todo identifiying the PR and version of iceberg that introduced the fileIOcloser and instructions on removing the code duplication later?